crates/subspace-networking/src/shared.rs

```diff
@@ -103,7 +103,7 @@ pub(crate) enum Command {
         result_sender: oneshot::Sender<Vec<PeerId>>,
     },
     Bootstrap {
-        result_sender: mpsc::UnboundedSender<()>,
```

Comment:
The type has an implicit semantics (sync-async invocation). Please, add the comment at the very least.